webrtc.html

```diff
@@ -1289,97 +1283,110 @@
               </li>
 
               <li>
-                <p>Let <var>p</var> be a new promise.</p>
-              </li>
-
-              <li>
-                <p>In parallel, start the process to apply
-                <var>candidate</var>.</p>
-
-                <div class="note">We probably want to define the process to
-                'apply an ICE candidate' in JSEP and link to it above.</div>
+                <p>Return the result of <a href=
+                "#enqueue-an-operation">enqueuing</a> the following steps:</p>
 
                 <ol>
                   <li>
-                    <p>If <var>candidate</var> is <code>null</code>, the User
-                    Agent MUST queue a task that runs the following steps:</p>
-                    <ol>
-                      <li>
-                        <p>For each <a>media description</a> in the last
-                        successfully applied remote description, perform the
-                        processing for an end-of-candidates indication for
-                        said <a>media description</a> as defined in
-                        [[TRICKLE-ICE]].</p>
-                      </li>
-                      <li>
-                        <p>Resolve <var>p</var> with <code>undefined</code>.</p>
-                      </li>
-                    </ol>
+                    <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                    <code>true</code>, return a promise rejected with an
+                    <code>InvalidStateError</code>.</p>
                   </li>
 
                   <li>
-                    <p>If <var>candidate</var> could not be successfully,
-                    applied the User Agent MUST queue a task that runs the
-                    following steps:</p>
-
-                    <ol>
-                      <li>
-                        <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot
-                        is <code>true</code>, then abort these steps.</p>
-                      </li>
-
-                      <li>
-                        <p>Reject <var>p</var> with a <code>DOMException</code>
-                        object whose <code>name</code> attribute has the value
-                        <code>OperationError</code> and abort these steps.</p>
-                      </li>
-                    </ol>
+                    <p>Let <var>p</var> be a new promise.</p>
                   </li>
 
                   <li>
-                    <p>If <var>candidate</var> is applied successfully, the
-                    User Agent MUST queue a task that runs the following
-                    steps:</p>
+                    <p>In parallel, start the process to apply
+                    <var>candidate</var>.</p>
```

Comment:
We're "calling" a JSEP-process here and waiting for the result. Compare with the step 5.1 in [1].

[1] https://www.w3.org/2001/tag/doc/promises-guide#example-add-bookmark